runtime/queries/rust/highlights.scm

```diff
@@ -49,6 +49,14 @@
 (loop_label
   (identifier) @type)
```

Comment:
While we're at it, how do we feel about changing this to
```scheme
(loop_label
  "'" @label
  (identifier) @label)
```
To match the lifetime labels? I was about to make a PR for this change but it might as well be included in this one instead if possible.